Apps/Inc/Contactor.h

```diff
@@ -3,12 +3,13 @@
 #ifndef CONTACTOR_H
 #define CONTACTOR_H
 
-#include "BSP_PWM.h"
-#include "Simulator.h"
-#include "common.h"
+#include "BSP_PWM.h" //PWM_Period is defined in here
 
 //Parameters for PWM
-#define PWM_ON_TIME 2000 //This number is determined through testing
+#define PWM_ON_TIME 3500
+
+#include "Simulator.h"
+#include "common.h"
```

Comment:
These lines didnt have to move. Only thing that should have changed is the PWM_ON_TIME